include/wlr/types/wlr_output.h

```diff
@@ -55,7 +55,7 @@ struct wlr_output {
 	struct wl_global *wl_global;
 	struct wl_list wl_resources;
 
-	char name[16];
+	char name[24];
```

Comment:
That one is because of HEADLESS-<maxint> being too large, could also check for the snprintf return value instead.

```
../backend/headless/output.c: In function ‘wlr_headless_add_output’:
../backend/headless/output.c:124:65: error: ‘%d’ directive output may be truncated writing between 1 and 11 bytes into a region of size 7 [-Werror=format-truncation=]
  snprintf(wlr_output->name, sizeof(wlr_output->name), "HEADLESS-%d",
                                                                 ^~
../backend/headless/output.c:124:55: note: directive argument in the range [-2147483647, 2147483647]
  snprintf(wlr_output->name, sizeof(wlr_output->name), "HEADLESS-%d",
                                                       ^~~~~~~~~~~~~
../backend/headless/output.c:124:2: note: ‘snprintf’ output between 11 and 21 bytes into a destination of size 16
  snprintf(wlr_output->name, sizeof(wlr_output->name), "HEADLESS-%d",
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   wl_list_length(&backend->outputs) + 1);
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Note: this 16 comes from the EDID spec.

Reply:
I'll revert it and check the return value then. I don't see much use of having > 999999 HEADLESS outputs...

Reply:
On the contrary

Reply:
wl_outputs have nothing to do with EDID

Reply:
Fair enough, less work if I don't have to do anything :)

Reply:
Then we shouldn't have fixed-size fields?

Reply:
nah it' sk

Reply:
My bad, this is the only fixed-sized field that isn't part of EDID.

Reply:
I'm pretty sure I set it to 16 because it was big enough to fit all of the DRM names with a reasonable number of outputs (i.e. not having 4 billion) and it was a nice round number.